pkg/storage/modules/copy_on_write_test.go

```diff
@@ -149,3 +151,74 @@ func TestCopyOnWriteReadOverrunNonMultiple(t *testing.T) {
 
 	assert.Equal(t, onlydata[size-50:], buff2[:50])
 }
+
+func TestCopyOnWriteCRCIssue(t *testing.T) {
+
+	// Create a new block storage, backed by memory storage
+	size := 16 * 1024
+	blockSize := 64 * 1024
+
+	fstore, err := sources.NewFileStorageSparseCreate("test_data_sparse", uint64(size), blockSize)
+	assert.NoError(t, err)
+
+	t.Cleanup(func() {
+		os.Remove("test_data_sparse")
+	})
+
+	//	fstore := sources.NewMemoryStorage(size)
```

Comment:
```suggestion
```
It seems like a leftover?

Reply:
454758083ba76bb024b7b6a8d91aba7d2df057aa